packages/flutter/lib/src/material/dialog.dart

```diff
@@ -556,12 +557,15 @@ class SimpleDialogOption extends StatelessWidget {
   /// Typically a [Text] widget.
   final Widget child;
 
+  /// The amount of space by which to inset the [children].
```

Comment:
Add documentation on what the default value would be if this weren't provided (ie. "If null, then... is used.")